docs/standard/analyzers/api-analyzer.md

```diff
@@ -13,16 +13,28 @@ ms.technology: dotnet-standard
 
 The .NET API Analyzer is a Roslyn analyzer that discovers potential compatibility risks for C# APIs on different platforms and detects calls to deprecated APIs. It can be useful for all C# developers at any stage of development.
 
-API Analyzer comes as a NuGet package [Microsoft.DotNet.Analyzers.Compatibility](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/) and after you reference it in a project, it automatically monitors the code and indicates problematic API usage. You can also get suggestions on possible fixes by clicking on the light bulb. The drop-down menu includes an option to suppress the warnings.
+API Analyzer comes as a NuGet package [Microsoft.DotNet.Analyzers.Compatibility](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/). After you reference it in a project, it automatically monitors the code and indicates problematic API usage. You can also get suggestions on possible fixes by clicking on the light bulb. The drop-down menu includes an option to suppress the warnings.
 
 > [!NOTE]
 > The .NET API analyzer is still a pre-release version.
 
+## Prerequisites
+
+* Visual Studio 2017 or Roslyn 2.0
```

Comment:
we talked about this section offline. Not clear what Roslyn 2.0 means here. Does it work VS for Mac? Does it work with any kind of project?